### PR TITLE
64 order new timelines desc

### PR DESCRIPTION
**Pull request by berggren**
Order the list of latest timelines in desc order.

Ref issue: #64 
Reviewer: @onager

### CODE REVIEW

File: timesketch/ui/templates/sketch/timelines.html [hunk 1]
```diff
@@ -20,40 +20,42 @@
...
-        </form>
+      {% if form.timelines.choices %}
+          <div class="row">
+            <div class="col-md-12">
+              <div class="card">
+                <h4>Last 20 timelines added</h4>
+                <form method="post" action="{{ url_for('sketch_views.timelines', sketch_id=sketch.id) }}">
+                  {{ form.timelines }}
+                  {{ form.csrf_token }}
+                  {% if form.timelines.choices %}
```
Comment (Onager) [inline timesketch/ui/templates/sketch/timelines.html]: Isn't this a duplicate of the if statement this is already inside?

File: timesketch/ui/views/sketch.py [hunk 2]
```diff
@@ -157,7 +159,7 @@ def timelines(sketch_id):
         indices = indices.filter(SearchIndex.name.contains(query)).limit(500)
         filtered = True
     if not filtered:
-        indices = indices.limit(10)
+        indices = indices.limit(20)
```
Comment (Onager) [inline timesketch/ui/views/sketch.py]: Can you pull this magic number out into a constant somewhere?

File: timesketch/ui/templates/sketch/timelines.html [hunk 3]
```diff
@@ -20,40 +20,42 @@
...
-          {{ form.csrf_token }}
-          {% if form.timelines.choices %}
-          <button class="btn btn-primary" type="submit">Add to sketch</button>
-          {% endif %}
-        </form>
+      {% if form.timelines.choices %}
+          <div class="row">
+            <div class="col-md-12">
+              <div class="card">
+                <h4>Last 20 timelines added</h4>
```
Comment (Onager) [inline timesketch/ui/templates/sketch/timelines.html]: I think it's better to not hardcode this number - pull it out into a var somewhere else.

File: timesketch/ui/templates/sketch/timelines.html (hunk 1 above)
Comment (berggren) [inline timesketch/ui/templates/sketch/timelines.html]: Yes, fixed now.

File: timesketch/ui/views/sketch.py (hunk 2 above)
Comment (berggren) [inline timesketch/ui/views/sketch.py]: Done.

File: timesketch/ui/templates/sketch/timelines.html (hunk 3 above)
Comment (berggren) [inline timesketch/ui/templates/sketch/timelines.html]: Done.

**Comment by Onager** [comment]
LGTM

**Pull request merged**